README.md

```diff
@@ -1,75 +1,81 @@
-# QHub
+# `qhub` - automated data science environments on cloud environments
 
-Is a tool for initializing and maintaining the state of QHub
-deployments on Digital Ocean, Amazon Web Services, and Google Cloud
-Platform.
+:badges:
```

Comment:
I guess this is a placeholder, which needs to filled (or removed) before merging this PR.

Reply:
i'm going to add these badges in a few min. thanks for looking out